app/models/dqt_trn_request.rb

```diff
@@ -20,14 +20,17 @@
 class DQTTRNRequest < ApplicationRecord
   belongs_to :application_form
 
-  enum :state, { pending: "pending", complete: "complete" }, default: :pending
-
+  enum :state,
+       { initial: "initial", pending: "pending", complete: "complete" },
+       default: :initial
   validates :state, presence: true, inclusion: { in: states.values }
 
   def update_from_dqt_response(response)
     if (trn = response[:trn]).present?
```

Comment:
Should this just passing in a TRN? I don't think we should be parsing the response in the model. I don't like the idea of passing nil in here either though, or moving the state logic into the worker.

Reply:
It could be moved into a service class, but that feels like overkill to me. I think I don't mind passing `nil` in if we name the method correctly.